pkg/scheduler/internal/queue/testing.go

```diff
@@ -25,6 +25,11 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 )
 
+// NewTestQueue creates a priority queue with an empty informer factory.
```

Comment:
Revert to the original place

Reply:
My intension here was to read the code from up to down.